pkg/config/loader.go

```diff
@@ -49,12 +50,12 @@ func LoadConfig() (*Config, error) {
 
 	// 1 - Read in the config file.
 	if err = viper.ReadInConfig(); err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
```

Comment:
For most of the base level operations that are not too deep do we need the WithStack?  

Other areas make a lot of sense, but what's the motivation for stack traces at this level?